lib/connections/service.go

```diff
@@ -129,17 +129,18 @@ type service struct {
 	*suture.Supervisor
 	connectionStatusHandler
 
-	cfg                  config.Wrapper
-	myID                 protocol.DeviceID
-	model                Model
-	tlsCfg               *tls.Config
-	discoverer           discover.Finder
-	conns                chan internalConn
-	bepProtocolName      string
-	tlsDefaultCommonName string
-	limiter              *limiter
-	natService           *nat.Service
-	evLogger             events.Logger
+	cfg                    config.Wrapper
+	deviceAddressesChanged chan struct{}
```

Comment:
The order of the early variables in the struct matches those of passed in parameters, I'd put this further down (maybe after the `evLogger`) as it's an internally constructed thing.